scripts/generateDocs.js

```diff
@@ -75,7 +75,7 @@ console.log(FORMATS_PATH + ' generated.');
 
 const TEMPLATES_PATH = './docs/templates.md'
 const templates = jsdoc2md.renderSync({
-  files: ['lib/common/templates.js'],
+  files: ['lib/common/formats.js'],
```

Comment:
We should just remove this whole code block from 76 to 83 because it is no longer doing anything important, it is just taking a handlebars file and turning it into a markdown file. Lets just check in the markdown file and not generate it anymore.